staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -18,6 +26,12 @@
           <mesh filename="package://staubli_rx160_support/meshes/rx160/collision/base_link.stl"/>
         </geometry>
       </collision>
+      <inertial>
+          <mass value="${base_mass}"/>
+          <inertia ixx="0.001" ixy="0.0" ixz="0.0"
+                   iyy="0.001" iyz="0.0"
+                   izz="0.001"/>
+        </inertial>
```

Comment:
I agree with you, but unless Staubli provides us with the real values, or someone takes apart the robots and get the values empirically, this is indeed as good as it gets.

I did not claim them to be the real values, just actual values extracted from official CAD models using well-known and respectable CAD software. ;-)

You are right, a disclaimer in the package description is a sensible approach. I'll amend to my last commit!

BTW, the hard work with Solidworks was courtesy of @psotiropoulos.